src/mixins/filing-template-mixin.ts

```diff
@@ -81,6 +84,25 @@ export default class FilingTemplateMixin extends Vue {
         shareClasses[index].series = share.series?.filter(x => x.action !== ActionTypes.REMOVED)
           .map((x) => { const { action, ...rest } = x; return rest })
       }
+
+      // Filter out and modify name translation
+      const translations = this.stateModel.nameTranslations as NameTranslationDraftIF[]
```

Comment:
This code looks fine but please break it out into a new function. THIS method should do 1 thing but it has grown :)

(Maybe the other filters above should be functioned out too.)